### PR TITLE
Executive 2021 03 05

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/votes/Executive vote - March 5, 2021.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote. 
+
+---
+
+## Executive Summary
+
+If this executive proposal passes the following **changes** will occur within the Maker Protocol:
+- The ETH-A [Target Available Debt](https://community-development.makerdao.com/en/learn/governance/module-dciam) (`gap`) will increase from 30 million DAI to 80 million DAI.
+- The Bid Duration (ttl) parameter will decrease from 6 hours to 4 hours for all vault types where liquidations are enabled.
```
Comment (0xElihu) [inline governance/votes/Executive vote - March 5, 2021.md:17]: change ttl to `ttl`

File: governance/votes/Executive vote - March 5, 2021.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- The Max Auction Duration (tau) parameter will decrease from 6 hours to 4 hours for all vault types where liquidations are enabled.
```
Comment (0xElihu) [inline governance/votes/Executive vote - March 5, 2021.md:18]: change tau to `tau`

File: governance/votes/Executive vote - March 5, 2021.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- The `box` parameter will increase from 15 million DAI to 20 million DAI.
+- The Minimum Bid Increment (beg) parameter will increase from 3% to 5% for the ETH-B vault type.
```
Comment (0xElihu) [inline governance/votes/Executive vote - March 5, 2021.md:20]: change beg to `beg`

File: governance/votes/Executive vote - March 5, 2021.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- The Minimum Bid Increment (beg) parameter will increase from 3% to 5% for the UniswapV2 Liquidity Pool vault types.
```
Comment (0xElihu) [inline governance/votes/Executive vote - March 5, 2021.md:21]: change beg to `beg`

File: governance/votes/Executive vote - March 5, 2021.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+### Onboarding RWA-001
+
+As per this [poll](https://vote.makerdao.com/polling/QmSqXVUQ?network=mainnet#poll-detail), this executive proposal represents the latest step in onboarding RWA-001 (6s Capital). If this executive proposal passes the RWA001-A vault type will be added using RWA001 (6s Capital) as collateral with preliminary, non-final parameters for testing purposes.
+
+The preliminary parameters for RWA-001 (6s Capital) are:
+
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 3%
+* [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 1000 DAI
+
+Please review the following forum threads containing information about RWA-001 (6s Capital) to inform your position before voting.
```
Comment (0xElihu) [inline governance/votes/Executive vote - March 5, 2021.md:57]: Does it make sense to add MIP21 here? https://forum.makerdao.com/t/mip21-real-world-assets-off-chain-asset-backed-lender/3917

**Review by derek-flossman — APPROVED** [review]
Nice, new format lgtm!

**Review by 0xElihu — CHANGES_REQUESTED** [review]

**Pull request merged**